engine/src/w32printer.cpp

```diff
@@ -1360,6 +1362,8 @@ void MCWindowsPrinter::DoFinalize(void)
 	m_dc_changed = false;
 
 	MCValueRelease(m_name);
+	// SN-2015-03-18: [[ Win shutdown crash ]] Do not over-release m_name
+	m_name = NULL;
```

Comment:
How does setting this to NULL stop an over-release? After 'DoFinalize()' is called, the printer object isn't again used until DoInitialize() is - or is it?